dev/ScrollViewer/ScrollViewer.idl

```diff
@@ -79,6 +79,8 @@ unsealed runtimeclass ScrollViewer : Windows.UI.Xaml.Controls.Control
     [MUX_DEFAULT_VALUE("ScrollViewer::s_defaultAnchorRatio")]
     [MUX_PROPERTY_VALIDATION_CALLBACK("ValidateAnchorRatio")]
     Double VerticalAnchorRatio { get; set; };
+    void RegisterAnchorCandidate(Windows.UI.Xaml.UIElement element);
```

Comment:
what is the scenario ? I thought the anchor requested event was the way to achieve this.

Reply:
The idea was to expose the same methods at the ScrollViewer level as the Scroller level (without implementing the interface in both places though), as a convenient pass through to the inner Scroller control. It is true that handling the AnchorRequested event lets you populate the candidates too. Let's check it in and confirm with Micah when he comes back.